### ISSUE TITLE
How to I set Black Notepad as default Notepad

### ISSUE THREAD

**Issue by Smartx38** [savaged/BlackNotepad#19]
When I installed Black Notepad and opened it everything is fine but i want to set it to my default Notepad

**Comment by eprosdev** [comment]
Yeah I can't figure out how to do this either. It doesn't show up as an option when I try to add it

**Comment by Chispasito** [comment]
Reposting my comment from another thread:

It is possible to make it the default text editor. It is a long process due to the microsoft store permissions. If you do not have admin rights, this will not be possible for you.

I'm also making a short-story long, so bear with me. Simply put, make yourself the owner of both the WindowsApps folder and the blacknotepad app, then you change the default text app to blacknotepad. More details below:

The first thing you're going to need is permissions to access the location of Black Notepad. For me, Black Notepad's default location was in C:\Program Files\WindowsApps\55809savaged.BlackNotepad_1.0.9.0_neutral__p2aarvyam9pfc\BlackNotepad\

However, you cannot jump the gun and try to access that location because WindowsApps is a secure folder owned by the system. Let's change that

Unhide all folders so you can find the WindowsApps folder within Program Files
Right click on WindowsApps -> select properties -> go to security tab -> click advanced
We need to remove "system" as the owner. At the top left, select "change". You may need admin rights for this.
This section will ask for an "object" to select. This means whichever people you want to have access to this must be defined. Note, I have not tried limiting this to admin. I input my personal path. I would recommend doing the same, or keeping it general at "users"
a) If you are unsure of what names you can enter, go to your c drive, then "users" folder and look at the names listed there.
Select check names, then "ok" once it confirms your entry.
Another personal option here is to replace the owner on all subfolders. I did so, but it may still work without doing this step. This is because the subfolders within windowsapps don't have extra security
Now that we have access to windowsapps, we need to change the owner on the blacknotepad app itself.

C:\Program Files\WindowsApps\55809savaged.BlackNotepad_1.0.9.0_neutral__p2aarvyam9pfc\BlackNotepad\

Find the location of the blacknotepad application. For me, it was in the path listed above. But if you're hunting for it, just click on all the blacknotepad folders until you find one that says BlackNotepad. Make sure to copy your specific path as you might need it later
Once you get to the .exe, right-click on it, and select properties -> security tab -> advanced
Change the owner to your preferred profile(s)
Now that we have given ourselves the appropriate permissions, go to a .txt file -> select it -> right-click -> "Open with" -> "choose another app" -> ensure to check "always use this app to open .txt files"

If black notepad doesn't show up on the short list, click on more apps, and then "look for another app on this pc". This will open a file explorer where you can then paste the path you copied earlier to easily find the black notepad.exe location.

______
This is what I remember for how to do so. If your experience is different from what I have above, please let me know so people can have the most accurate information on how to do this.
